src/components/carousel/index.js

```diff
@@ -60,6 +60,10 @@ export default class Carousel extends BaseComponent {
      * callback for onScroll event of the internal ScrollView
      */
     onScroll: PropTypes.func,
+    /**
+     * Should the container be animated (send the animation style via containerStyle)
+     */
+    animate: PropTypes.bool,
```

Comment:
keep the name consistent with the prop we have for the `View` and `Text` components. 
```suggestion
    animated: PropTypes.bool,
```

Reply:
Fixed